src/MockXhrFactory.js

```diff
@@ -0,0 +1,17 @@
+var MockXhr = require('./MockXhr');
```

Comment:
I'm not sure if you ant to add this file to the MockXhr.js file.

Also, I have not added this to the package exports because I'm not sure how you want that handled so that the require statement is not changed.